UnityProject/Assets/Scripts/Managers/MatrixManager/MatrixManager.cs

```diff
@@ -279,7 +279,7 @@ private static MatrixInfo CreateMatrixInfoFromMatrix(Matrix matrix, int id)
 	public static MatrixInfo AtPoint(Vector3Int worldPos, bool isServer, MatrixInfo possibleMatrix = null)
 	{
 		//for performance, this is just a suggestion on which matrix we believe this point could be in, not always correct
-		if (possibleMatrix != null && IsInMatrix(worldPos, isServer, possibleMatrix))
+		if (possibleMatrix is null && IsInMatrix(worldPos, isServer, possibleMatrix))
```

Comment:
``is null`` is the opposite of ``!= null``

Reply:
its fixed in the next commit but now im reverting it anyways